fiatconnect-api.md

```diff
@@ -2233,6 +2234,41 @@ A KYC schema containing personal data about a user, as well as documents such as
 
 The `selfieDocument` and `identificationDocument` fields should be base64 encoded binary blobs representing images.
 
+#### 9.3.1.2. `PersonalDataAndDocumentsWithBack`
+
+A brasilian standard KYC schema containing personal data about a user, as well as documents such as an ID, photo and selfie.
+The accepted documents are [RG](https://en.wikipedia.org/wiki/Brazilian_identity_card), [CNH](https://en.wikipedia.org/wiki/Driving_licence_in_Brazil) and [RNM](https://en.wikipedia.org/wiki/Registro_Nacional_de_Estrangeiros)
```

Comment:
all 3 of these documents have CPF on them, which the image would presumably include. Does idNumber still need to be its own field? Seems like it could be redundant, though I don't know the details of how you're doing KYC checks..

Reply:
if you can drop the idNumber field, or clarify that it means these things specifically in the case where some field like `documentType` is equal to RG, CNH, or RNM, then you could potentially make this schema more general / reusable across regions

Reply:
which could be helpful for other providers and consumers of the FC API